### PR TITLE
docs: helm doc improvements

### PR DESCRIPTION
**Pull request by keon94**

### CODE REVIEW

File: docs/QuickStart/HelmSetup.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ helm install devlake . \
     --set "ingress.tlsSecretName=ssl-certificate"
 ```
 
-After deployed, visit devlake: https://devlake-0.example.com:8443, and grafana at https://devlake-0.example.com:8443/grafana
```
Comment (hezyin) [inline docs/QuickStart/HelmSetup.md]: @keon94 Are you sure we don't need the port number here? If that's the case, do we also want to update line 56 of this file?

Comment (keon94) [reply docs/QuickStart/HelmSetup.md]: Honestly, having played with it some more, the ingress.HttpsPort setting has been inconsequential on my machine. I used both 8443 and 8444. Turns out 8443 is coincidentally the target port on the kube apiserver, and if I try hitting it directly I get a forbidden error. But 8444 maps to nothing, which leads me to believe this is not doing anything.

**Comment by Startrekzky** [comment]
Hey @keon94 , can you solve the conflicts here? Thanks

**Comment by Startrekzky** [comment]
Hi @keon94 , the CI failed. Can you try `npm run build` on your local machine to fix it?

**Comment by Startrekzky** [comment]
Hi @keon94 , can you solve the conflicts if you still want it to be merged? If not, I'll close it.

**Pull request closed without merging**